lib/stdlib/doc/src/erl_pp.xml

```diff
@@ -146,6 +152,28 @@
           but only for the guard test <c><anno>Guard</anno></c>.</p>
       </desc>
     </func>
+
+    <func>
+      <name name="legalize_vars" arity="1" since=""/>
+      <fsummary>Ensure all variable names are valid.</fsummary>
+      <desc>
+        <p>The Erlang compiler will, when expanding records to tuples,
+          introduce new variables in the abstract representation. As the
+          expansion is done on the abstract representation, the compiler
+          can safely name the new variables with names that are not
+          syntactically valid in Erlang source code (The name starts
```

Comment:
```suggestion
          syntactically valid in Erlang source code (the name starts
```

(You might want do the same correction in the commit message.)

Reply:
fixed.